src/main/java/com/mael/ttt/Game.java

```diff
@@ -45,7 +43,7 @@ private void start() {
     }
 
     private void setPlayers() {
-        List<Player> players = new Menu(gameUI).createPlayers();
+        List<Player> players = new MenuCreator().createMenu(gameUI).createPlayers();
```

Comment:
It is very strange that a `Menu` has the responsibility of the actual creation of a `Player` (_or rather a `List<Player>`_).

Isn't this a direct violation of  *SRP*?

- [wikipedia] https://en.wikipedia.org/wiki/Single_responsibility_principle
- [pdf] [Object Mentor article](https://www.google.co.uk/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=0ahUKEwjy_oGiwsnJAhUNhhoKHeCdBF8QFgglMAE&url=http%3A%2F%2Fwww.objectmentor.com%2Fresources%2Farticles%2Fsrp.pdf&usg=AFQjCNHQQ1Aw-2yCciEbERpJn3VdHBCmQw)